lib/jsonapi/config.ex

```diff
@@ -3,16 +3,14 @@ defmodule JSONAPI.Config do
   Configuration struct containing JSON API information for a request
   """
 
-  alias JSONAPI.Page
-
   defstruct data: nil,
             fields: %{},
             filter: [],
             include: [],
             opts: nil,
             sort: nil,
             view: nil,
-            page: %Page{}
+            page: %{}
```

Comment:
100% agree that this is the right choice now that I understand this portion of the spec a little better.